README.md

```diff
@@ -352,6 +351,51 @@ Private properties should be declared in class extensions (anonymous categories)
 @end
 ```
 
+## Method Ordering
+
+Method implementations are grouped and ordered by class inheritance, then by protocol conformance with system protocols proceeding custom protocols. A `pragma mark` is used to separate each group.
```

Comment:
> with system protocols proceeding custom protocols

I am not sure I agree with specifying this explicitly. Often my custom protocols feel, for lack of a better term, more "important" or more "domain-specific" than the UIKit et al. protocols, which tend to be more generic and feel more like boilerplate (obviously generalizing here).

Aside from this, I'm on board with this PR; @mattbischoff, what would you think about eliminating this one phrase? I don't think that change diminishes the value of the guideline.